.github/actions/setup-node/action.yml

```diff
@@ -40,8 +40,11 @@ runs:
       with:
         timeout_minutes: 10
         max_attempts: 3
-        command: yarn install --frozen-lockfile --ignore-scripts
+        command: |
+          yarn install --frozen-lockfile --ignore-scripts
+          yarn prepare:install-re2
+          yarn prepare:re2
 
-    - name: Run scripts
+    - name: Generate files
       shell: bash
-      run: yarn prepare
+      run: test -d lib && yarn prepare:generate || exit 0
```

Comment:
🤔 why `exit 0`? which false errors can happen here?

Reply:
When we do sparse checkout, there is no `lib` folder, so `test -d lib` has error status, but we don't want it to make the entire action fail.